eos/b-decays/observables.cc

```diff
@@ -378,13 +379,13 @@ namespace eos
             R"(The option "l" selects the charged lepton flavor.)"
             R"(The option "form-factors" selects the form factor parametrization.)",
             {
-                make_observable("B_s->Klnu::dBR/dq2", R"(d\mathcal{B}(\bar{B}_s\to K\ell^-\bar\nu)/dq^2)",
+                make_observable("B_s->Klnu::dBR/dq2", R"(d\mathcal{B}(\barB_s\to K\ell^-\bar\nu)/dq^2)",
```

Comment:
What's with this change everywhere?

Reply:
It is a typo, thanks. I had to rename many decays and I overlooked the fact that I messed up :)